crates/ruff_linter/src/settings/defaults.rs

```diff
@@ -72,7 +73,15 @@ pub static INCLUDE: Lazy<Vec<FilePattern>> = Lazy::new(|| {
 
 impl Default for Settings {
     fn default() -> Self {
+        let project_root = path_dedot::CWD.clone();
         Self {
+            cache_dir: cache_dir(&project_root),
```

Comment:
Nit: maybe a comment on why these are separated from the rest? Same with other instantiations.

Reply:
I'll leave it as is because these will be moved to their own `struct` in a follow-up PR, making the reason clear